### ISSUE TITLE
[Feature request]: toRgbRatio

### ISSUE THREAD

**Issue by lukasoppermann** [scttcper/tinycolor#227]
Hey, in some cases, e.g. when delivering data to iOS you need a ratio rgb color. It would be great if tinycolor could also support `toRgbRatio`.

I think it would work like this

```js
const color = new TinyColor('red');
color.toRgbRatio(); // { r: 1, g: 0, b: 0, a: 1 }
```

It should be pretty similar to `toPercentageRgb` since the ratio is basically a percentage value as a float from `0` to `1`.

**Comment by matthias-ccri** [comment]
Agreed this is needed, because `toRgb` rounds values, which loses precision. So there's actually no way to get the _exact_ RGB values out! 

See https://github.com/bgrins/TinyColor/issues/278
